src/queries/services/rewards/postCreateActivity.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+import { postCreateActivityModelCreator } from 'queries/models/rewards';
+import { CreateActivityPayload } from 'queries/types';
+// import { appConfig } from 'config';
+
+export const createActivity = (payload: CreateActivityPayload) =>
```

Comment:
```suggestion
export const createActivity = (payload: CreateActivityPayload) =>
```

@abhishek-01k naming